orchestrator/relayer/valset_relaying.go

```diff
@@ -69,7 +68,28 @@ func (s *peggyRelayer) RelayValsets(ctx context.Context, currentValset *types.Va
 				return err
 			}
 
-			s.logger.Info().Str("tx_hash", txHash.Hex()).Msg("sent Ethereum Tx (EthValsetUpdate)")
+			//TODO: estimate gas and profitability using "valset reward" param.
+			//
+			// Ref: https://github.com/umee-network/peggo/issues/56
+
+			// Checking in pending txs (mempool) if tx with same input is already submitted
+			// We have to check this at the very last moment because any other relayer could have submitted
```

Comment:
```suggestion
			// Checking in pending txs (mempool) if tx with same input is already submitted.
			// We have to check this at the very last moment because any other relayer could have submitted.
```